frontend/src/Components/ContentComponent.tsx

```diff
@@ -2,6 +2,7 @@ import React, {useEffect, useRef, useState} from 'react';
 import classNames from 'classnames';
 import styles from './ContentComponent.module.scss';
 import {createObserverService, observeOnHidden} from '../Services/ObserverService';
+import * as Vimeo from '@vimeo/player';
```

Comment:
https://bundlephobia.com/package/@vimeo/player@2.18.0
21Kb gzipped - not horrible but so far we only had [1 vimeo link](https://orbitar.space/search?term=vimeo) so don't need to include it by default - if we can load it async - let's do that

Reply:
its just @types/vimeo__player for type safe compiling

Reply:
It may have unexpected effects of including `@vimeo/player` into bundle if we ever include it. You can explicitely instruct compiler that you only need types by using `import type`